datafusion/functions-aggregate-common/src/aggregate/groups_accumulator/accumulate.rs

```diff
@@ -351,6 +260,106 @@ impl NullState {
     }
 }
 
+pub fn accumulate<T, F>(
```

Comment:
Could you please add some documentation to this function that explains what it does and its arguments?

Reply:
Done in https://github.com/apache/datafusion/pull/12095/commits/0f1b64ca75898cb0f179b70e1b55f953b306a858